tools/libaqr/deployment.py

```diff
@@ -250,7 +293,7 @@ def _gen_vagrantfile(
             f"""
     config.vm.define :"node{nid}", primary: {is_primary_str} do |node|
         node.vm.hostname = "node{nid}"
-        node.vm.network "forwarded_port", guest: 1337, host: {host_port}, host_ip: "*"
+        node.vm.network "forwarded_port", guest: 1337, host: {host_port}
```

Comment:
(pr not related) I did not know that we hardcode 1337 port.

Reply:
@AvengerMoJo Why did you do this, I didn't double check this? It breaks functionality of open ports.